From 3c32647ce5cb9485adb7fb165d731512172f9135 Mon Sep 17 00:00:00 2001
Date: Wed, 25 Nov 2020 13:29:18 +0100
Subject: [PATCH] Increase buffer size.

---
 lldb/source/Target/Platform.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lldb/source/Target/Platform.cpp b/lldb/source/Target/Platform.cpp
index aaf48f35f92..bba5d688d29 100644
--- a/lldb/source/Target/Platform.cpp
+++ b/lldb/source/Target/Platform.cpp
@@ -1644,7 +1644,7 @@ Status Platform::DownloadModuleSlice(const FileSpec &src_file_spec,
     return error;
   }
 
-  std::vector<char> buffer(1024);
+  std::vector<char> buffer(1024 * 100);
   auto offset = src_offset;
   uint64_t total_bytes_read = 0;
   while (total_bytes_read < src_size) {
-- 
2.25.1.windows.1

